src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -293,16 +293,20 @@ export const SavedSearchAlertEditFormQueryRenderer: React.FC<SavedSearchAlertEdi
       variables={{
         id: editAlertEntity.id,
       }}
-      placeholder={<SavedSearchAlertEditFormPlaceholder />}
+      placeholder={
+        <SavedSearchAlertEditFormPlaceholder onCloseClick={onCloseClick} />
+      }
       cacheConfig={{ force: true }}
       render={({ props, error }) => {
         if (error) {
           console.error(error)
           return null
         }
 
-        if (!props?.me?.alert || !props?.viewer) {
-          return <SavedSearchAlertEditFormPlaceholder />
+        if (!props || !props?.me?.alert || !props?.viewer) {
```

Comment:
This should be equivalent: 
```suggestion
        if (!props?.me?.alert || !props?.viewer) {
```